wandb/sdk/data_types/image.py

```diff
@@ -148,12 +149,23 @@ def __init__(
         if isinstance(data_or_path, Image):
             self._initialize_from_wbimage(data_or_path)
         elif isinstance(data_or_path, str):
-            self._initialize_from_path(data_or_path)
+            if Image.path_is_reference(data_or_path):
+                self._path = data_or_path
+                self._is_tmp = False
+                self._sha256 = hashlib.sha256(data_or_path.encode("utf-8")).hexdigest()
+            else:
+                self._initialize_from_path(data_or_path)
+            ext = os.path.splitext(data_or_path)[1][1:]
+            self.format = ext
         else:
             self._initialize_from_data(data_or_path, mode)
 
         self._set_initialization_meta(grouping, caption, classes, boxes, masks)
 
+    @classmethod
+    def path_is_reference(cls, path: str) -> bool:
```

Comment:
This seems to be a copy from image. Perhaps we can refactor this to be a utility function since it is cls independent?